rotkehlchen/chain/aggregator.py

```diff
@@ -544,6 +544,9 @@ def query_balances(
                 xpub_manager.check_for_new_xpub_addresses(blockchain=blockchain)  # type: ignore # is checked in the if
         else:  # all chains
             for chain in SupportedBlockchain:
+                if chain.is_evm() and len(self.accounts.get(chain)) == 0:
```

Comment:
why only for evm?

Reply:
eth2 is an exception since we don't have a balance structure for them. Left only that one unchecked